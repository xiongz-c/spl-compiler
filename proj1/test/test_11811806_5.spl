char 1_err;

int -:)() {
    float 0fxs = 2.0;
    char 123 >= 2;
    char a = 'c'
}