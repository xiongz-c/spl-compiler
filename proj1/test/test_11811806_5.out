Error type A at Line 1: unknown lexeme 1_err
Error type A at Line 3: unknown lexeme :
Error type A at Line 4: unknown lexeme 0fxs
Error type B at Line 4: Definition not match rules
Error type B at Line 5: Definition not match rules
Error type B at Line 6: Missing semicolon';'
